### DISCUSSION TITLE
Where to find  deepin 20.9's kernel code (kernel version 15.18.17)?

### DISCUSSION THREAD

**Discussion by Bottle-sys** [linuxdeepin/developer-center#7548] [Development Questions | 开发答疑] [answered]
I need to rebuild kernel code to clarify some HW issues. After installing Deep In version 20.9 which is formal released, I can see it's kernel verion is 15.18.17. Is this version of kernel code exactly algined with github code tag 15.18.17（https://github.com/deepin-community/kernel/tree/v5.18.17/）？

**Comment by BLumia** [comment, marked as answer]
I'm not quite sure about v20.9 since I don't maintain packages, but you might want to try use `apt source <package-name>` to get the matching source code of the package. I also suggest join the kernel Matrix chatroom to discuss with the deepin kernel maintainers directly if possible :)

**Reply by Bottle-sys** [reply]
Thank you for your rely!  And do you know where to join the kernel Matrix chatroom?

**Reply by BLumia** [reply]
[As suggested earlier](https://github.com/orgs/linuxdeepin/discussions/7514#discussioncomment-8809499), you can join the `#deepin-kernel:deepin.org` Matrix room. If you are not familiar with Matrix, [Mozilla's this wiki page](https://wiki.mozilla.org/Matrix) could help a lot.
